### DISCUSSION TITLE
ostree install progress (Writing objects)

### DISCUSSION THREAD

**Discussion by champtar** [rhinstaller/anaconda#4588] [Ideas]
When installing an ostree system, progress message just says "Writing objects" without any number to show actual progress.
It would be nice to show some information showing progress (number of objects written / size / ...)

**Comment by VladimirSlavik** [comment]
@jkonecny12 is currently working on some changes to ostree, and IIRC mentioned that detailed progress might not be possible.

**Reply by champtar** [reply]
Even just the number of objects written without the total would get you a feeling of how fast the progress is (better than nothing), but if it needs changes in both rpm-ostree and anaconda I understand it might get complicated

**Reply by jkonecny12** [reply]
Hi, sorry for late reply. I was changing just the new container solution for RPM which AFAIK should be future solution of the deployment of ostree. However, we don't have any reasonable way of progress reporting (could be added later). So the situation there is even worse because we don't know what is happening.

About the current ostree, I honestly don't know exactly what is possible there. It's some time I was looking on this, might be possible. Unfortunately, we don't have time now to look on this improvement :(. Could be something we can look on later.
